docs/samples/v1beta1/transformer/torchserve_image_transformer/README.md

```diff
@@ -142,3 +142,25 @@ Handling connection for 8080
 * Connection #0 to host localhost left intact
 {"predictions": [2]}
 ```
+
+## Testing with a local transformer & predictor in Kubernetes (developer guide)
```

Comment:
I think this is super useful! We should move this to a more general local developer guide for custom transformer/predictor.

Reply:
in kerve/website?